libraries/Microsoft.Bot.Builder.Core/TurnContext.cs

```diff
@@ -211,61 +211,64 @@ public bool Responded
         /// <remarks>If the activities are successfully sent, the task result contains
         /// an array of <see cref="ResourceResponse"/> objects containing the IDs that 
         /// the receiving channel assigned to the activities.</remarks>
-        public async Task<ResourceResponse[]> SendActivities(IActivity[] activities)
+        public Task<ResourceResponse[]> SendActivities(IActivity[] activities)
```

Comment:
Same null checking comments. Public methods on public classes, need to check.

Reply:
✔️